### ISSUE TITLE
php undefined function imagecreatetruecolor()

### ISSUE THREAD

**Issue by tattwamasii2p** [wipedlifepotato/webwallets#2]
```
2021-05-16T05:32:15.087471+00:00 heroku[router]: at=info method=GET path="/style/css/keyframes.css" host=equinoxstaging.herokuapp.com request_id=79866f92-a9ae-4ccc-850c-853dcc8dcba4 fwd="x.x.x.x" dyno=web.1 connect=1ms service=1ms status=200 bytes=1505 protocol=https
2021-05-16T05:32:15.238860+00:00 app[web.1]: [16-May-2021 05:32:15 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function imagecreatetruecolor() in /app/fincore/wwnl-php/sys/captcha.php:25
2021-05-16T05:32:15.238890+00:00 app[web.1]: Stack trace:
2021-05-16T05:32:15.239391+00:00 app[web.1]: #0 /app/fincore/wwnl-php/sys/captcha.php(88): Simply_Captcha_Render->initImage()
2021-05-16T05:32:15.239556+00:00 app[web.1]: #1 /app/fincore/wwnl-php/init_captcha.php(5): Simply_Captcha->__construct()
2021-05-16T05:32:15.239560+00:00 app[web.1]: #2 {main}
2021-05-16T05:32:15.239717+00:00 app[web.1]: thrown in /app/fincore/wwnl-php/sys/captcha.php on line 25
2021-05-16T05:32:15.239985+00:00 app[web.1]: [16-May-2021 05:32:15 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function imagecreatetruecolor() in /app/fincore/wwnl-php/sys/captcha.php:25
2021-05-16T05:32:15.240024+00:00 app[web.1]: Stack trace:
2021-05-16T05:32:15.240164+00:00 app[web.1]: #0 /app/fincore/wwnl-php/sys/captcha.php(88): Simply_Captcha_Render->initImage()
2021-05-16T05:32:15.240346+00:00 app[web.1]: #1 /app/fincore/wwnl-php/init_captcha.php(7): Simply_Captcha->__construct()
2021-05-16T05:32:15.240413+00:00 app[web.1]: #2 {main}
2021-05-16T05:32:15.240549+00:00 app[web.1]: thrown in /app/fincore/wwnl-php/sys/captcha.php on line 25
2021-05-16T05:32:15.240831+00:00 heroku[router]: at=info method=GET path="/init_captcha.php?fake=true&i=3" host=equinoxstaging.herokuapp.com request_id=cfa1e306-abae-463a-8ca8-527ce3a336f8 fwd="x.x.x.x" dyno=web.1 connect=1ms service=3ms status=500 bytes=264 protocol=https

```

**Comment by tattwamasii2p** [comment]
```
2021-05-16T05:32:15.240831+00:00 heroku[router]: at=info method=GET path="/init_captcha.php?fake=true&i=3" host=equinoxstaging.herokuapp.com request_id=cfa1e306-abae-463a-8ca8-527ce3a336f8 fwd="x" dyno=web.1 connect=1ms service=3ms status=500 bytes=264 protocol=https
2021-05-16T05:32:15.240859+00:00 heroku[router]: at=info method=GET path="/init_captcha.php?fake=true&i=1" host=equinoxstaging.herokuapp.com request_id=98ea22af-2bc2-40c4-9ec1-766e4f30fd07 fwd="x" dyno=web.1 connect=1ms service=2ms status=500 bytes=264 protocol=https
2021-05-16T05:32:15.240917+00:00 app[web.1]: [16-May-2021 05:32:15 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function imagecreatetruecolor() in /app/fincore/wwnl-php/sys/captcha.php:25
2021-05-16T05:32:15.240924+00:00 heroku[router]: at=info method=GET path="/init_captcha.php?fake=true&i=2" host=equinoxstaging.herokuapp.com request_id=56c27d23-649f-4368-8cfe-a0edbbcbd9e5 fwd="x" dyno=web.1 connect=1ms service=2ms status=500 bytes=264 protocol=https
2021-05-16T05:32:15.240962+00:00 app[web.1]: Stack trace:
2021-05-16T05:32:15.240973+00:00 heroku[router]: at=info method=GET path="/init_captcha.php" host=equinoxstaging.herokuapp.com request_id=abe1ac5c-939c-49ef-810d-200987bdb99d fwd="x" dyno=web.1 connect=0ms service=3ms status=500 bytes=264 protocol=https
2021-05-16T05:32:15.241104+00:00 app[web.1]: #0 /app/fincore/wwnl-php/sys/captcha.php(88): Simply_Captcha_Render->initImage()
2021-05-16T05:32:15.241226+00:00 app[web.1]: #1 /app/fincore/wwnl-php/init_captcha.php(5): Simply_Captcha->__construct()
2021-05-16T05:32:15.241254+00:00 app[web.1]: #2 {main}
2021-05-16T05:32:15.241372+00:00 app[web.1]: thrown in /app/fincore/wwnl-php/sys/captcha.php on line 25
2021-05-16T05:32:15.241881+00:00 app[web.1]: x- - [16/May/2021:05:32:15 +0000] "GET /init_captcha.php?fake=true&i=3 HTTP/1.1" 500 - "https://equinoxstaging.herokuapp.com/style/css/captcha.css" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0
2021-05-16T05:32:15.242410+00:00 app[web.1]: x - - [16/May/2021:05:32:15 +0000] "GET /init_captcha.php HTTP/1.1" 500 - "https://equinoxstaging.herokuapp.com/style/css/captcha.css" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0
2021-05-16T05:32:15.242708+00:00 app[web.1]: [16-May-2021 05:32:15 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function imagecreatetruecolor() in /app/fincore/wwnl-php/sys/captcha.php:25
2021-05-16T05:32:15.242739+00:00 app[web.1]: Stack trace:
2021-05-16T05:32:15.242888+00:00 app[web.1]: #0 /app/fincore/wwnl-php/sys/captcha.php(88): Simply_Captcha_Render->initImage()
2021-05-16T05:32:15.243015+00:00 app[web.1]: #1 /app/fincore/wwnl-php/init_captcha.php(5): Simply_Captcha->__construct()
2021-05-16T05:32:15.243018+00:00 app[web.1]: #2 {main}
2021-05-16T05:32:15.243129+00:00 app[web.1]: thrown in /app/fincore/wwnl-php/sys/captcha.php on line 25
2021-05-16T05:32:15.243588+00:00 app[web.1]: x - - [16/May/2021:05:32:15 +0000] "GET /init_captcha.php?fake=true&i=1 HTTP/1.1" 500 - "https://equinoxstaging.herokuapp.com/style/css/captcha.css" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0

```

**Comment by tattwamasii2p** [comment]
fixed at commit 981864f7ecdd347ae960ee517e2fbdc08e943fe
